frontend/src/metabase/qb/components/actions/CommonMetricsAction.jsx

```diff
@@ -16,9 +16,10 @@ export default ({ question }: ClickActionProps): ClickAction[] => {
     }
 
     const activeMetrics = query.table().metrics.filter(m => m.isActive());
+    const mname = <strong>{metric.name}</strong>
     return activeMetrics.slice(0, 5).map(metric => ({
         name: "common-metric",
-        title: <span>View <strong>{metric.name}</strong></span>,
+        title: <span>{t`View  ${ mname }`}</span>,
```

Comment:
You can convert this to a variable but you can also use `jt` to handle jsx tags. It looks something like this

```jsx
<span>{ jt`View ${<span>{metric.name}</span>}`}</span>
```

[Here's an example](https://github.com/metabase/metabase/pull/6278/files#diff-9881a49f8bf82cd653f982fbb650c80bR181) from our alerts feature branch.